webpack.config.js

```diff
@@ -49,6 +49,28 @@ module.exports = {
       },
       {
         test: /\.css$/,
+        exclude: [
+          path.resolve(__dirname, 'src/css/styles.css'),
+          path.resolve(__dirname, 'src/css/diff.css')
```

Comment:
I think it would be extremely helpful to add a comment explaining why these files are special/excluded, and that we aim to eventually remove them.